### ISSUE TITLE
Fatal failure at src/theory/bv/bitblast/lazy_bitblaster.cpp:227

### ISSUE THREAD

**Issue by muchang** [cvc5/cvc5#5396]
For this formula, cvc4 throws out a fatal failure:
```
[514] % cvc4 -q small.smt2
Fatal failure within virtual void CVC4::theory::bv::TLazyBitblaster::bbTerm(CVC4::TNode, CVC4::theory::bv::TLazyBitblaster::Bits&) at src/theory/bv/bitblast/lazy_bitblaster.cpp:227
Check failure
 node.getType().isBitVector()
Aborted
[515] % cat small.smt2
(declare-fun a () Int)
(assert (= (bv2nat (bvor ((_ int2bv 3) a) ((_ int2bv 3) a))) 0))
(assert (distinct ((_ extract 0 0) (bvsdiv ((_ int2bv 3) (bv2nat (bvmul ((_ int2bv 3) a) ((_ int2bv 3) a)))) ((_ int2bv 3) 1))) (_ bv0 1)))
(check-sat)
[516] %
```
Commit: d301b51

**Comment by rainoftime** [comment]
Another formula
~~~~
(declare-fun v1 () (_ BitVec 4))
(assert (distinct (bv2nat (bvudiv v1 v1)) 0))
(check-sat)
~~~~

**Comment by mpreiner** [comment]
Fixed by #6660.
